src/win32/setup-syscheck.c

```diff
@@ -13,11 +13,16 @@
 #include "../error_messages/error_messages.h"
 #include <errno.h>
 #define OSSEC_CONFIG_TMP  ".tmp.ossec.conf"
-
+#include "dll_load_notify.h"
 
 /* Enable Syscheck */
 int main(int argc, char **argv)
 {
+#ifdef WIN32
```

Comment:
Why the compilation conditional is added here (same as in other .c files inside src/win32)?
That a file is inside the directory src/win32 doesn't guarantee that is for windows compilation?

Reply:
I'll check this, it might there unnecessarily.
Thank you.